pkg/api/router.go

```diff
@@ -1,21 +1,100 @@
 package api
 
 import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/darchlabs/api-example/pkg/person"
 	"github.com/julienschmidt/httprouter"
 )
 
+// Define response structure
 type response struct {
-	Data interface{} `json:"data,omitempty"`
-	Meta interface{} `json:"meta,omitempty"`
+	Data  interface{} `json:"data,omitempty"`
```

Comment:
remove this spaces

Reply:
Which spaces?

`// Define response structure
type response struct {
	Data  interface{} `json:"data,omitempty"`
	Meta  interface{} `json:"meta,omitempty"`
	Error interface{} `json:"error,omitempty"`
}`

Here it is how I have those lines of codes right now.